src/main/java/com/hackclub/hccore/slack/SlackBot.java

```diff
@@ -76,7 +76,7 @@ public class SlackBot implements Listener {
   private final HCCorePlugin plugin;
   private final SocketModeApp socket;
   private final String commandBase;
-  private final HashMap<String, String> mcUuidVerificationCodes = new HashMap<>();
+  private final HashMap<UUID, String> mcUuidVerificationCodes = new HashMap<>();
```

Comment:
Maybe it could be offline player, but not required at all

Reply:
eh, I think just the UUID is fine, if an offline player is needed it can always be fetched from the server

Reply:
Yep ok :+1: